frontend/src/stories/TeamsTable.stories.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import TeamsTable from "../components/regions/UserDetails/TeamsTable/Presentation";
-import user from "./fixtures/user.json";
```

Comment:
please make use of the user AND the authUser. They have different purposes. Both are relevent